sdks/java/harness/src/test/java/org/apache/beam/fn/harness/FnApiDoFnRunnerTest.java

```diff
@@ -164,7 +166,7 @@
 @SuppressWarnings({
   "rawtypes", // TODO(https://issues.apache.org/jira/browse/BEAM-10556)
 })
-public class FnApiDoFnRunnerTest implements Serializable {
```

Comment:
Why drop serializable?

Reply:
Was messing with some stuff attempting to get the test working. Forgot to re-add. Though as an aside I'm not sure why this is necessary since everything seems fine with out.